src/problem1.rb

```diff
@@ -0,0 +1,9 @@
+class Problem1
+  def sum(array)
+    unless array.nil?
+      array.sum
```

Comment:
Yes indeed, sorry, looks like i need new glasses, I will solve the problem asap.